api/provider/aws/test_aws/helpers.go

```diff
@@ -52,3 +55,55 @@ func deleteSGHelper(mockAWS *awsc.MockClient, securityGroupID string) {
 		DeleteSecurityGroup(input).
 		Return(&ec2.DeleteSecurityGroupOutput{}, nil)
 }
+
+func healthCheckHelper(healthCheck *models.HealthCheck) *elb.HealthCheck {
+	if healthCheck == nil {
```

Comment:
this block needs to be removed - too much 'magic'. The caller should be responsible for passing in the default